cmd/ike/session/session_suite_test.go

```diff
@@ -13,3 +15,10 @@ func TestSession(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecWithJUnitReporter(t, "Session Suite")
 }
+
+var _ = SynchronizedAfterSuite(func() {}, func() {
+	goleak.VerifyNone(GinkgoT(),
```

Comment:
At this point, one can start to wonder about that one 'global' suite..

Reply:
The only problem is that you can only have one "Before/AfterSuite" hook per suite, so we would need to put all the logic there, does not matter if it's used by every test or not. A bit of a disadvantage.